registry/consumer/organization_event_consumer.py

```diff
@@ -56,8 +56,13 @@ def _get_org_details_from_blockchain(self, event):
         ipfs_org_metadata = self._ipfs_util.read_file_from_ipfs(org_metadata_uri)
         owner = blockchain_org_data[3]
         members = blockchain_org_data[4]
+        self._remove_owner_from_members(members, owner)
```

Comment:
can members ever be None?

Reply:
nope. it is in contract that function return signature will be (bool, bytes, bytes, str, list, list, list)